build.rs

```diff
@@ -110,12 +110,24 @@ fn render_file(path: PathBuf, sql_type: &SqlType) -> Result<String, Error> {
     let mut result = String::new();
     let mut paths: Vec<PathBuf> = fs::read_dir(path)?.map(|f| f.unwrap().path()).collect();
     paths.sort();
+    let mut prev = 0;
     paths.into_iter().for_each(|path| {
+        let curr = numeric_prefix(&path);
+        // assert that the numeric prefixes are in order with no gaps or duplicates
+        assert_eq!(curr, prev + 1);
+        prev = curr;
         result += wrap(&path, sql_type).as_str();
     });
     Ok(result)
 }
 
+/// Parses and returns the numeric prefix in the filename of each idempotent and migration file
+fn numeric_prefix(path: &Path) -> i32 {
+    let file_name = path.file_name().unwrap().to_str().unwrap();
+    let (prefix, _) = file_name.split_once('-').unwrap();
+    prefix.parse::<i32>().unwrap()
```

Comment:
Since someone could conceivably forget to put a number in front of their migration name I suggest failing with an explicit message instead of `unwrap()`.

Reply:
will do

Reply:
fixed!